server/game/cards/13.5-TB/Sparrows.js

```diff
@@ -0,0 +1,54 @@
+const DrawCard = require('../../drawcard');
+
+class Sparrows extends DrawCard {
+    setupCardAbilities() {
+        this.reaction({
+            when: {
+                onCardOutOfShadows: event => event.card === this && this.opponentHasPower()
+            },
+            target: {
+                cardCondition: card => (
+                    card.location === 'play area' &&
+                    card !== this &&
+                    card.hasTrait('The Seven'))
+            },
+            handler: context => {
+                let opponents = this.game.getOpponents(this.controller).filter(opponent => opponent.faction.power > 0);
+
+                if(opponents.length === 0) {
+                    return;
+                }
+
+                if(opponents.length === 1) {
+                    this.stealPowerFromOpponent(opponents[0], context.target);
+                    return;
+                }
+
+                let factionCards = opponents.map(opponent => opponent.faction);
+
+                this.game.promptForSelect(this.controller, {
+                    activePromptTitle: 'Select a faction card',
+                    cardCondition: card => factionCards.includes(card),
+                    cardType: 'faction',
+                    onSelect: (player, card) => this.stealPowerFromOpponent(card.owner, context.target)
+                });
+            }
+        });
+    }
+
+    stealPowerFromOpponent(opponent, targetCard) {
+        this.game.addMessage('{0} moves 1 power from {1}\'s faction card to {2}',

```

Comment:
Messages should include the originating card (e.g. 'Player 1 uses Sparrows to move 1 power from Player 2's faction card to Sept of Baelor')